docs/sql_reference/src/asciidoc/_chapters/sql_functions_and_expressions.adoc

```diff
@@ -150,67 +150,109 @@ See the individual entry for the function.
 == Datetime Functions
 
 These functions use either a datetime value expression as an argument or
-return a result of datetime data type:
+return a result of DATETIME, VALUE or CHAR:
 
-[cols="25%,75%"]
+[caption="Table 7-3 "]
+.Overview of DATETIME Functions
+[cols="25%,50%,25%"]
 |===
+| *Name* | *Description* | *Return Data Type*
 | <<add_months_function,ADD_MONTHS Function>>                               | Adds the integer number of months specified by _intr_expr_
 to _datetime_expr_ and normalizes the result.
+| DATE
 | <<converttimestamp_function,CONVERTTIMESTAMP Function>>                   | Converts a Julian timestamp to a TIMESTAMP value.
+| TIMESTAMP
 | <<current_function,CURRENT Function>> | Returns the current timestamp. You can also use the
 <<current_timestamp_function,CURRENT_TIMESTAMP Function>>.
+| TIMESTAMP
 | <<current_date_function,CURRENT_DATE Function>>                           | Returns the current date.
+| DATE
 | <<current_time_function,CURRENT_TIME Function>>                           | Returns the current time.
+| TIME
 | <<current_timestamp_function,CURRENT_TIMESTAMP Function>> | Returns the current timestamp. You can also use the <<current_function,CURRENT Function>>.
+| TIMESTAMP
 | <<date_add_function,DATE_ADD Function>>                                   | Adds the interval specified by _interval_expression_
 to _datetime_expr_.
+| DATE
 | <<date_part_function_of_an_interval,DATE_PART Function (of an Interval)>> | Extracts the datetime field specified by _text_ from the interval value
 specified by interval and returns the result as an exact numeric value.
+| NUMERIC VALUE
 | <<date_part_function_of_a_timestamp,DATE_PART Function (of a Timestamp)>> | Extracts the datetime field specified by _text_ from the datetime value
 specified by timestamp and returns the result as an exact numeric value.
+| NUMERIC VALUE
 | <<date_sub_function,DATE_SUB Function>>                                   | Subtracts the specified _interval_expression_ from
 _datetime_expr._
+| DATE or TIMESTAMP
 | <<date_trunc_function,DATE_TRUNC Function>>                               | Returns the date with the time portion of the day truncated.
+| TIMESTAMP
 | <<dateadd_function,DATEADD Function>>                                     | Adds the interval specified by _datepart_ and _num_expr_
 to _datetime_expr_.
+| DATE or TIMESTAMP
 | <<datediff_function,DATEDIFF Function>>                                   | Returns the integer value for the number of _datepart_ units of time
 between _startdate_ and _enddate_.
+| INTEGER VALUE
```

Comment:
On the other hand, "INTEGER" is a noun as well as an adjective. So, "INTEGER" (without VALUE) seems better.

Reply:
@DaveBirdsall Thanks for your correction, I've changed "INTEGER VALUE" to "INTERGER".